tests/source/test_bazefetcher.py

```diff
@@ -112,3 +112,36 @@ def test_no_time_zone():
     with pytest.raises(ValueError) as excinfo1: baze('Perlin', t1_1, t1_3_notz)
     assert 'dates must be timezone aware' in str(excinfo0.value)
     assert 'dates must be timezone aware' in str(excinfo1.value)
+
+def test_extend_range_backward_within_file():
+    cos_b = baze('Cos-T60s-SR1hz',
```

Comment:
The authored/installation-04-status tag is intended for testing these kinds of data